Tasks/FtpUpload/ftpuploadtask.ts

```diff
@@ -0,0 +1,243 @@
+/// <reference path="../../definitions/vsts-task-lib.d.ts" />
+/// <reference path="../../definitions/node.d.ts" />
+/// <reference path="../../definitions/Q.d.ts" />
+
```

Comment:
More feedback coming this afternoon ...

Get Outlook for iOS<https://aka.ms/o0ukef>




On Fri, Jul 15, 2016 at 12:31 PM -0400, "David Staheli" <notifications@github.com<mailto:notifications@github.com>> wrote:


In Tasks/FtpUpload/ftpuploadtask.ts<https://github.com/Microsoft/vsts-tasks/pull/2087#discussion_r71001677>:

> @@ -0,0 +1,243 @@
> +/// <reference path="../../definitions/vsts-task-lib.d.ts" />
> +/// <reference path="../../definitions/node.d.ts" />
> +/// <reference path="../../definitions/Q.d.ts" />
> +


Can you organize all of the global declarations and statements to the top of the file and all of the functions below that?

-
You are receiving this because you were assigned.
Reply to this email directly, view it on GitHub<https://github.com/Microsoft/vsts-tasks/pull/2087/files/6cc41c7763674244984cdc87ad8395b3c5e66ed4#r71001677>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AA4IDM6jYEoor_5C3veZenIIz0hRwaXbks5qV7XlgaJpZM4JNjxX>.